cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -473,7 +473,7 @@ data:
         retry_forever
         retry_max_interval 30
         chunk_limit_size 2M
-        queue_limit_length 8
+        total_limit_size 2G
```

Comment:
this seems quite large for a default install.  I wouldn't want this to go past 500MB.

How did you arrive at this setting?

Reply:
OK，then set total_limit_size 500M can achieve this goal